AMP-109903 Update docs for data permission restrictions (#339)
amplitude/amplitude-docs: content/collections/data/en/amplitude-data-settings.md

For any unplanned events, event properties, event property types, user properties, or user property types, you can tell Amplitude Data to either mark them as **unexpected**, or to **reject** them outright. Amplitude Data ingests any events or properties marked as unexpected and send a notification to everyone subscribed to this schema. If you choose to reject unexpected data, however, Amplitude Data doesn't ingest or store the rejected data. Subscribers still receive a notification.

Click _Save_ to implement any changes you make to your schema settings.

> ## The Permissions Tab

Reviewer: ```suggestion
## The Permissions tab
```

> Your company may want to limit the number of people who have access to perform Data Management functions. For example, you may want most of your users to have the Member role so they can create dashboards and charts but prefer that custom event creation and other data management features be limited to managers or administrators.

Reviewer: ```suggestion
You may want to limit the number of people who have access to perform Data Management functions. For example, if you want most of your users to have the Member role so they can create dashboards and charts but limit custom event creation and other data management features to managers or administrators.
```

In the permissions tab, you can add additional restrictions for selected roles, limiting data management access while keeping other permissions for those roles.

> Under Restrict access in Amplitude Data, you can specify from the following options to restrict permissions to the viewer level on a per-project basis:

Reviewer: ```suggestion
Under Restrict access in Amplitude Data, specify from the following options to restrict permissions to the viewer level on a per-project basis:
```
@cyhk Is this in Data settings? Will it be a new tab?

Author: Yes, this will be a new tab in Data settings.

Author: To clarify - the Permissions tab is a new tab in Data settings, Restrict access in Amplitude Data is the title of the bottom section on the page, which looks like this:
<img width="1034" alt="Screenshot 2024-10-18 at 16 50 34" src="https://github.com/user-attachments/assets/3fa11bc3-9cea-439c-9f40-79201688e1d0">

- Use Default Restrictions
- Restrict Members
- Restrict Members and Managers

Then, click _Save_.

> The Permissions tab is only visible to admins and permission restrictions cannot be applied to admins themselves.

Reviewer: ```suggestion
The Permissions tab is only visible to users with the Administrator role, and permission restrictions don't apply to Administrators.
```

> ### Copy to Other Projects

Reviewer: ```suggestion
### Copy to other projects
```

> You can also copy current permission restriction settings to other projects by clicking on the _Copy to Other Projects_ button. To copy settings, select the desired projects and click _Save_.

Reviewer: ```suggestion
Click _Copy to Other Projects_ to apply the current permission restriction settings to another project.
```
